docs/02-app/02-api-reference/05-next-config-js/images.mdx

```diff
@@ -88,6 +89,17 @@ export default function cloudfrontLoader({ src, width, quality }) {
   return url.href
 ```
 
+### Azion
+
+Before using this loader, make sure to enable the Image Processor module in your Azion application following this guide: [Image Processor first steps](https://www.azion.com/en/documentation/products/guides/image-processor/).
+
+```js
+// Docs: https://www.azion.com/en/documentation/products/build/edge-application/image-processor/
+export default function azionLoader({ src, width, quality}) {
+  return `https://example.com/${src}?ims=${width}x/filters:quality(${quality} || 75)`
```

Comment:
This looks wrong since the docs say `ims` must come last 

> To ensure proper processing of images, the ims= query string parameter must be the last parameter in the URL. If any additional query string parameters are included after ims=, the request may return a 504 error.

Reply:
Hey, thanks for the feedback. 

I reviewed this doc to make sure (found an error on 75 fallback), but this seems correct to me. The `ims` is the only parameter in the URL. When [combine multiple filters](https://www.azion.com/en/documentation/products/build/edge-application/image-processor/#combine-multiple-filters), it looks like this.